lib/data_update_scripts/20200326145114_re_index_feed_content_to_elasticsearch.rb

```diff
@@ -0,0 +1,28 @@
+module DataUpdateScripts
+  class ReIndexFeedContentToElasticsearch
+    def run
+      clear_existing_feed_documents
+
+      index_docs(Article.pluck(:id), "Article")
+      index_docs(PodcastEpisode.pluck(:id), "PodcastEpisode")
+      index_docs(Comment.pluck(:id), "Comment")
```

Comment:
I think this is okay but, maybe we can query in batches to avoid loading too many IDs (articles are hundreds of thousand, don't know how many comments, millions?) in one pass?

Reply:
We have 384k comments. Pluck is really fast since we don't have many records for any of these tables and I want to make sure this job finishes without getting killed by another deploy. The select I found is really slow.